### PR TITLE
Editor: Preview pure skeleton.

### PR DESCRIPTION
**Pull request by gonnavis**
Let editor can preview pure skeleton file, which has not SkinnedMesh.
Such as some animation pack files downloaded from https://www.mixamo.com/ :
[Sword and Shield Pack.zip](https://github.com/mrdoob/three.js/files/7561408/Sword.and.Shield.Pack.zip)

### CODE REVIEW

File: editor/js/Editor.js [hunk 1]
```diff
@@ -424,6 +424,10 @@ Editor.prototype = {
 
 					helper = new THREE.SkeletonHelper( object.skeleton.bones[ 0 ] );
 
+				} else if ( object.isBone && ( !object.parent || !object.parent.isBone ) ) {
```
Comment (Mugen87) [inline editor/js/Editor.js:427]: I think this can be simplified like so:
```js
} else if ( object.isBone && ( object.parent?.isBone !== true ) ) {
```

**Comment by Mugen87** [comment]
https://raw.githack.com/gonnavis/three.js/EditorPreviewPureSkeleton/editor/index.html

File: editor/js/Editor.js (hunk 1 above)
Comment (Mugen87) [reply editor/js/Editor.js:427]: BTW: The policy is to test for `is*` flags like so:
```js
console.log( object.isBone === true ); // detecting all bones
console.log( object.isBone !== true ); // detecting all 3D object except for bones
```

Comment (gonnavis) [reply editor/js/Editor.js]: Thanks! Updated.

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**